stglib/hobo.py

```diff
@@ -59,6 +64,8 @@ def csv_to_cdf(metadata):
 
     metadata.pop("skiprows")
     metadata.pop("skipfooter")
+    if "ncols" in metadata:
```

Comment:
Can you add details to the docs about any new features (e.g. in config yaml) and instrument types supported with this PR?